### ISSUE TITLE
exception when setting postal in java client

### ISSUE THREAD

**Issue by metasnw** [metasfresh/metasfresh#2851]
### Is this a bug or feature request?
bug

### What is the current behavior?
cant set postal in java client

#### Which are the steps to reproduce?

```
Ausnahme:
Column not updateable: Postal - NewValue=87301 - OldValue=8730; Note to developer: to bypass this checking you can1. Set AD_SysConfig 'org.compiere.PO.C_Location_Postal.ThrowExIfNotUpdateable' = 'N' to disable this exception (will still be logged with Level=SERVERE)2. Set dynamic attribute ModelDynAttributeAccessor {
attributeName = org.adempiere.model.InterfaceWrapperHelper#ReadOnlyColumnCheckDisabled attributeType = class java.lang.Boolean } = true (no errors will be logged in this case)

```

**Comment by teosarca** [comment]
i think that's happening because C_Location.Postal column has IsUpdateable=N.

**Comment by metasnw** [comment]
addressed via #2933
